centos/6/Dockerfile

```diff
@@ -1,25 +1,38 @@
 FROM centos:6
 MAINTAINER Roman Tsisyk <roman@tarantool.org>
 
+# The only available devtoolset-7 in Vault archives found, check:
+#   https://vault.centos.org/centos/6/sclo/x86_64/rh/Packages/d/
+ARG DEVTOOLSET=devtoolset-7
+
+# Removed missing CentOS-Base repositories
+ADD CentOS-Base.repo /etc/yum.repos.d/
+
 # Fix missing locales
 ENV LC_ALL="en_US.UTF-8" LANG="en_US.UTF-8"
 
 # Enable extra repositories
+RUN yum -y update
 RUN yum -y install \
     wget \
     curl \
     pygpgme \
     yum-utils
-RUN yum -y install epel-release
+
+# Install epel-release
 ADD CentOS-SCLo-scl.repo /etc/yum.repos.d/
 ADD CentOS-SCLo-scl-rh.repo /etc/yum.repos.d/
+RUN yum -y update
```

Comment:
Twice `yum update` has no much sense, I guess `yum makecache` was meant.

Reply:
There are some updates comparing to package version installed in the base image. So I would keep `yum update` and mark the purpose explicitly.

Reply:
Changed as suggested.